template.json

```diff
@@ -11,7 +11,10 @@
     "macos": "",
     "windows": ""
   },
-  "download": {},
+  "download": {
```

Comment:
```
    "download": {
        "GitHub": {
            "project_slug": "swift-nav/esthri",
            "linux": "esthri-8.7.0-x86_64-unknown-linux-musl.tar.gz",
            "macos": "esthri-8.7.0-x86_64-apple-darwin.tar.gz",
            "windows": "esthri-8.7.0-x86_64-pc-windows-msvc.tar.gz"
        }
    },
    "architecture": {
        "macos_x86": "",
        "macos_arm64": "esthri-8.7.1-alpha.0+arm64+sitl_fix-aarch64-apple-darwin.tar.gz"
    },
```
in the test examples, looked like you wanted to keep the `download` field as well the `architecture` field
not sure if this was a mistake